homework_1/task_02/taxes/testCases_test.go

```diff
@@ -0,0 +1,94 @@
+package taxes_test
+
+import (
+	"code-cadets-2021/homework_1/task_02/taxes"
+	"math"
+)
+
+type testCase struct {
+	inputAmount float64
+	class       []taxes.TaxClass
+	outputTax   float64
+	error       bool
+}
+
+func getTestCases() []testCase {
```

Comment:
Every test case is using exactly the same tax classes. Add test cases with different tax classes - especially some which are not in the happy path. For example, one where tax classes are not sorted correctly, one where threshold are negative etc...